cmd/machine-config-controller/testrender.go

```diff
@@ -0,0 +1,153 @@
+package main
```

Comment:
it's ok since we're not compiling this anywhere but could we move this out of cmd :thinking: